worlds/jakanddaxter/regs/BoggySwampRegions.py

```diff
@@ -0,0 +1,170 @@
+from typing import List
+from BaseClasses import CollectionState, MultiWorld
+from .RegionBase import JakAndDaxterRegion
+from .. import JakAndDaxterOptions, EnableOrbsanity
+from ..Rules import can_fight, can_reach_orbs
+
+
+def build_regions(level_name: str, multiworld: MultiWorld, options: JakAndDaxterOptions, player: int) -> List[JakAndDaxterRegion]:
+
+    # This level is full of short-medium gaps that cannot be crossed by single jump alone.
+    # These helper functions list out the moves that can cross all these gaps (painting with a broad brush but...)
+    def can_jump_farther(state: CollectionState, p: int) -> bool:
+        return state.has_any({"Double Jump", "Jump Kick"}, p) or state.has_all({"Punch", "Punch Uppercut"}, p)
+
+    def can_jump_higher(state: CollectionState, p: int) -> bool:
+        return (state.has("Double Jump", p)
+                or state.has_all({"Crouch", "Crouch Jump"}, p)
+                or state.has_all({"Crouch", "Crouch Uppercut"}, p)
+                or state.has_all({"Punch", "Punch Uppercut"}, p))
+
+    # Orb crates and fly box in this area can be gotten with yellow eco and goggles.
+    # Start with the first yellow eco cluster near first_bats and work your way backward toward the entrance.
+    main_area = JakAndDaxterRegion("Main Area", player, multiworld, level_name, 23)
+    main_area.add_fly_locations([43])
+
+    # Includes 4 orbs collectable with the blue eco vent.
+    first_bats = JakAndDaxterRegion("First Bats Area", player, multiworld, level_name, 4)
+
+    first_jump_pad = JakAndDaxterRegion("First Jump Pad", player, multiworld, level_name, 0)
```

Comment:
Stylistic decision, but you could utilize the fact that this defaults to 0 (here and elsewhere throughout the region files)
```suggestion
    first_jump_pad = JakAndDaxterRegion("First Jump Pad", player, multiworld, level_name)
```

Reply:
It helped during the region mapping process to explicitly write out all the orb counts, even if they were 0. If it's alright, I'd like to keep it for future reference.